transcom/mymove
# B 20046 update prime email content int2

## [B-20046](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A950506)

## Summary

This PR updates the move submitted and counseled text from
“HomeSafe is required to contact you within 24 hours of receiving your move task order.”
to 
“HomeSafe is required to contact you within one Government Business Day.”

### How to test

1. Instead of runnning make server_run run this:
    `EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run `
2. Run make client_run as normal
4. Make a HHG move with an email you have access to
5. After signing and submitting the move; confirm that the email sent to you has the new text.
6. After logging  as a SC and edit and submit move; confirm that the email sent to you has the new text.

### Backend


## Screenshots
_**Subject: Your counselor has approved your move details**_
<img width="981" alt="Screenshot 2024-06-24 at 8 47 56 AM" src="https://github.com/transcom/mymove/assets/146969726/eeb922b9-a564-4a7f-bbf4-54c01119812c">

_**Subject: Thank you for submitting your move details**_
<img width="922" alt="Screenshot 2024-06-24 at 8 47 37 AM" src="https://github.com/transcom/mymove/assets/146969726/761a83b6-e893-4916-8d81-80a6bc0202a6">
